main.R

```diff
@@ -0,0 +1,26 @@
+library(dplyr)
+library(miceadds)
+library(readr)
+library(stringr)
+library(sbtools)
+library(whisker)
+
+outer_dir <- 'C:/Users/SLEUNG/Github/ds-pipelines-targets-1'
+
+print('source the following files:')
+for (dir in list('1_fetch', '2_process', '3_visualize')) {
+  setwd(file.path(outer_dir, dir, 'src'))
+  src_files = list.files(pattern='\\.R$')
```

Comment:
Although R lets you use `=` for assignment, most R programmers prefer `<=` and it's the norm in our group. Can you please change this so that it's more idiomatic for other team members? (RStudio has a shortcut ctrl+= for the assignment operator, which is a bit less of a pain to type than `<-`)

Reply:
I wanted to add a link to an R style guide. [This](https://style.tidyverse.org/) is almost certainly the most commonly used one, and is the one we use here (although our adherence to it is imperfect).